### PR TITLE
Remove redundant experimental overrides from jobs

### PR DESCRIPTION
**Pull request by maniSbindra**
This PR is to resolve the issue https://github.com/kubernetes/perf-tests/issues/1036.
Now the variables can be overriden using the CL2_ prefix instead of the [experimental overrides]( https://github.com/kubernetes/perf-tests/tree/master/clusterloader2/testing/load/experimental/overrides) .
In https://github.com/kubernetes/perf-tests/pull/1125 default values of all the variables referenced in [experimental overrides]( https://github.com/kubernetes/perf-tests/tree/master/clusterloader2/testing/load/experimental/overrides)  other than ENABLE_NETWORKPOLICIES  are set to true.

### CODE REVIEW

File: config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml [hunk 1]
```diff
@@ -966,12 +966,6 @@ presubmits:
         - --test-cmd-args=--testconfig=testing/access-tokens/config.yaml
         - --test-cmd-args=--testoverrides=./testing/experiments/enable_prometheus_api_responsiveness.yaml
         - --test-cmd-args=--testoverrides=./testing/experiments/use_simple_latency_query.yaml
-        - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_configmaps.yaml
```
Comment (mm4tt) [inline config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml]: We can only do that for master jobs, i.e. the ones checking out perf-tests repo from master branch, see line 943 (it uses release-1.16 where your changes are not present)

Comment (maniSbindra) [reply config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml]: Got it, thanks.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml [hunk 2]
```diff
@@ -169,13 +169,7 @@ periodics:
...
       - --test-cmd-args=--testconfig=testing/density/config.yaml
       - --test-cmd-args=--testconfig=testing/load/config.yaml
-      - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_configmaps.yaml
-      - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_daemonsets.yaml
-      - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_jobs.yaml
-      - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_pvs.yaml
-      - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_secrets.yaml
-      - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_statefulsets.yaml
-      - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_networkpolicies.yaml
+      - --env=CL2_ENABLE_NETWORKPOLICIES=true
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml]: nit: please sort flags alphabetically

Comment (maniSbindra) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml]: Thanks. I have moved the ``` - --env=CL2_ENABLE_NETWORKPOLICIES=true  ```  in sorted order

**Comment by wojtek-t** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**